data/groups.json

```diff
@@ -175,7 +173,7 @@
     "nobonus": [],
     "tooltip": "https://liskascend.com/",
     "color": "bg-purple text-light",
-    "data": ["anamix", "gregorst", "kylefromohio", "goforlisk", "nimbus", "moosty", "sdrpa", "roblad", "lisktrust", "slamper", "puredelta", "diamse"]
+    "data": ["anamix", "crypto_embassy", "diamse", "kylefromohio", "liskascend", "liskmagazine", "moosty", "nimbus", "sdrpa", "slamper", "southpaw439"]
```

Comment:
Not really what I see here:

https://liskascend.com/#team

Can someone from ascend comment on the correct list please?

Reply:
The website is in the process of being updated. Can confirm that gregorst, goforlisk, lisktrust, roblad and puredelta have been removed and that myself (southpaw439), liskmagazine, liskascend and crypto_embassy have been added. Your list is correct at this point in time

Reply:
Sourced this from their blog

- https://liskascend.com/2018/10/08/ascend-relaunches-with-formation-of-liskascend-delegate-and-alliance-with-sherwood-pool-2/
- https://liskascend.com/2018/10/10/new-ascend-member-welcome-liskmagazine/